Gems/WhiteBox/Code/Source/EditorWhiteBoxComponent.cpp

```diff
@@ -685,13 +685,13 @@ namespace WhiteBox
         return m_worldAabb.value();
     }
 
-    AZ::Aabb EditorWhiteBoxComponent::GetLocalBounds()
+    AZ::Aabb EditorWhiteBoxComponent::GetLocalBounds() const
     {
         AZ_PROFILE_FUNCTION(AzToolsFramework);
 
         if (!m_localAabb.has_value())
         {
-            auto& whiteBoxMesh = *GetWhiteBoxMesh();
+            auto& whiteBoxMesh = *const_cast<EditorWhiteBoxComponent*>(this)->GetWhiteBoxMesh();
```

Comment:
`CalculatedAabb` doesn't accept a `const` for whiteBoxMesh?

Reply:
GetWhiteBoxMesh isn't const. It's called all over the place. I cut off changes here to keep the PR in scope of shape, bounds, and intersection buses.